src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -218,6 +220,20 @@ public void bookRoom(Booking booking) {
         bookingList.add(booking);
     }
 
+
+    // to update accordingly when implementing billing system.
```

Comment:
We should standardize and use /** */

Reply:
I think /** */ is just used for docs. For normal comments, it's better to use //

Reply:
I left it as //, is that alright?